addons/binding/pom.xml

```diff
@@ -67,6 +67,7 @@
     <module>org.openhab.binding.squeezebox</module>
     <module>org.openhab.binding.systeminfo</module>
     <module>org.openhab.binding.systeminfo.test</module>
+    <module>org.openhab.binding.tankerkoenig</module>
```

Comment:
Had a look into the features file. Working in my binding in the correct alphabetical order will be problematic since the file isn't ordered completely.  At least the (rather old) one that is in the fork of dolic.